attributes/default.rb

```diff
@@ -25,19 +25,22 @@
 default['ms_dotnet']['v2']['version']         = '2.0.SP2'
 default['ms_dotnet']['v2']['include_patches'] = true
 default['ms_dotnet']['v2']['feature_source']  = nil
+default['ms_dotnet']['v2']['perform_reboot']  = false
 default['ms_dotnet']['v2']['package_sources'] = {}
 default['ms_dotnet']['v2']['require_support'] = false
 
 # .NET 3 attributes
 default['ms_dotnet']['v3']['version']         = '3.5.SP1'
 default['ms_dotnet']['v3']['include_patches'] = true
 default['ms_dotnet']['v3']['feature_source']  = nil
+default['ms_dotnet']['v3']['perform_reboot']  = false
```

Comment:
is there a reason to have on attribute per dotnet version instead of one for the whole cookbook?